src/factories/routePathLinkFactory.ts

```diff
@@ -33,7 +39,7 @@ class RoutePathLinkFactory {
             endNode: NodeFactory.createNode(link.solmuByLnkloppusolmu),
             positions: RoutePathLinkFactory.getPositions(link.geojson),
             isStartNodeTimeAlignmentStop: false,
-            id: 'new-12345',
+            id: RoutePathLinkFactory.getTemporaryRoutePathLinkId(),
```

Comment:
I think using increasing counter would be better. New-1, new-2, new-3, new-4, ...

Reply:
Sure, this I agree on.